### PR TITLE
[ASCII-1482] Fix APM status type error 

### PR DESCRIPTION
**Pull request by pgimalac**
### What does this PR do?

Update `mkHuman` and `mkHumanDuration` status functions to take any type (instead of `float64` only), and then cast to `float64`.

### Motivation

Under some specific conditions the status will contain the following errors:
```
executing "generalStatus.tmpl" at <.memstats.Alloc>: invalid value; expected float64
```

The issue comes from this line of the status template:
```
Mem alloc: {{humanize .memstats.Alloc}} bytes<br>
```

`humanize` expects a float, `Alloc` is really an int, but due to Marshalling + Unmarshalling it used to be converted to a float.
We changed the way the status is generated and now it is back to an int, which means we get this error. 

### Additional Notes

Reported through support. See AGENT-11334.

### Describe how to test/QA your changes

Check the trace-agent part of the status, it should not contain any error.

### CODE REVIEW

**Review by rtrieu — APPROVED** [review]
LGTM

File: releasenotes/notes/status-trace-section-typing-ccd984242ee6ce99.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix typing error while generating the trace-agent status.
```
Comment (sgnn7) [inline releasenotes/notes/status-trace-section-typing-ccd984242ee6ce99.yaml:11]:
```suggestion
    Fix type conversion error while generating the trace-agent status.
```

**Review by sgnn7 — APPROVED** [review]
One tiny comment but LGTM

File: releasenotes/notes/status-trace-section-typing-ccd984242ee6ce99.yaml (hunk 1 above)
Comment (pgimalac) [reply releasenotes/notes/status-trace-section-typing-ccd984242ee6ce99.yaml:11]: Fair 👍

**Pull request merged**